src/routers/axelar/Router.sol

```diff
@@ -33,21 +33,40 @@ interface AxelarGatewayLike {
         external;
 }
 
+interface CentrifugeGatewayLike {
+    function handle(bytes memory message) external;
+}
+
 contract ConnectorAxelarRouter is AxelarExecutableLike {
     using TypedMemView for bytes;
     // why bytes29? - https://github.com/summa-tx/memview-sol#why-bytes29
     using TypedMemView for bytes29;
     using ConnectorMessages for bytes29;
 
+    mapping(address => uint256) public wards;
+
     ConnectorLike public immutable connector;
```

Comment:
Should we be consistent with the naming and prefix our stuff with `Centrifuge*`? 

```suggestion
    CentrifugeConnectorLike public immutable connector;
```

Reply:
Actually, I kind of disagree with this convention, but could totally be convinced if someone felt more strongly. But regardless, if we were to start renaming everything to `CentrifugeConnector` and `CentrifugeXCMRouter`, etc, I think it belongs in a separate PR.

Reply:
My subjective opinion: `CentrifugeConnector` is the main contract and should have the full name. Other contracts are then kind of "subcontracts" and are called `Connector*` simply because `CentrifugeConnector*` becomes rather long.

Reply:
@offerijns @NunoAlexandre I agree for contract naming, but do either of you have a strong opinion on interface and variable naming?

Reply:
I tend to prefer the brevity of `ConnectorLike`, `GatewayLike`, `XCMRouterLike`, etc

Reply:
Let's go with the short versions, I like Jeroen's "subjective opinion" :p so the main contract goes as `CentrifugeConnector` and the rest won't contain `Centrifuge*` as a prefix like you are suggesting @AStox 👍